core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1084,7 +1080,9 @@ private void deleteAndJournal(LockedInodePath inodePath, DeleteOptions deleteOpt
     long opTimeMs = System.currentTimeMillis();
     deleteInternal(inodePath, false, opTimeMs, deleteOptions);
     DeleteFileEntry deleteFile = DeleteFileEntry.newBuilder().setId(fileId)
-        .setRecursive(deleteOptions.isRecursive()).setOpTimeMs(opTimeMs).build();
```

Comment:
what I had in mind was an integration test that:

1) writes a file to both alluxio and UFS
2) deletes the file with alluxio only set to true
3) restarts alluxio from journal
4) check that the ufs file still exists

before your fix, the test should be failing and after your fix the test should be passing ... makes sense?